src/components/popup-components/ManageFavouritesButton/ManageFavouritesButton.tsx

```diff
@@ -31,7 +37,8 @@ const ManageFavouritesButton = ({ popupId }) => {
 
       if (error) {
         console.error(error)
-        return setStatus("rejected")
+        setStatus("rejected")
+        return
```

Comment:
No real comments, but I'm curious why we're explicitly opting to move the return to a new line.

Is this just a personal preference or hoping to follow any particular convention or something?